docs/config.md

```diff
@@ -94,6 +94,31 @@ for more details.
 |----------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------|
 | `OTEL_PROPAGATORS`   | Comma-separated list of propagators. Supported options: `tracecontext`, `baggage`, `b3multi`, `b3`. See [the OpenTelemetry specification](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.14.0/specification/sdk-environment-variables.md#general-sdk-configuration) for more details. | `tracecontext,baggage` |
 
+## Samplers
+
+Samplers allows to control the noise and overhead introduced by OpenTelemetry.
+See [the OpenTelemetry specification](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.15.0/specification/sdk-environment-variables.md?plain=1#L45-L80)
+for more details.
+
+| Environment variable      | Description                                     | Default value           |
+|---------------------------|-------------------------------------------------|-------------------------|
+| `OTEL_TRACES_SAMPLER`     | Sampler to be used for traces \[1\]                   | `parentbased_always_on` |
+| `OTEL_TRACES_SAMPLER_ARG` | String value to be used as the sampler argument \[2\] |                         |
+
+\[1\]: Supported values are:
+
+- `always_on`,
+- `always_off`,
+- `traceidratio`,
+- `parentbased_always_on`,
+- `parentbased_always_off`,
+- `parentbased_traceidratio`,
+- `parentbased_jaeger_remote`.
```

Comment:
Do we still use jaeger-related stuff?

Reply:
I have forgot to remove it...

Removed.